modules/hosted-control-planes-monitoring-dashboard.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-control-planes-monitoring-dashboard_{context}"]
+= Creating monitoring dashboards for hosted clusters
+
+The HyperShift Operator can create or delete monitoring dashboards in the management cluster for each hosted cluster that it manages.
+
+[#hosted-control-planes-enable-dashboard]
+== Enabling monitoring dashboards
+
+To enable monitoring dashboards in a hosted cluster, complete the following steps:
+
+. Create the `hypershift-operator-install-flags` config map in the `local-cluster` namespace, being sure to specify the `--monitoring-dashboards` flag in the `data.installFlagsToAdd` section. For example:
+
++
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: hypershift-operator-install-flags
+  namespace: local-cluster
+data:
+  installFlagsToAdd: "--monitoring-dashboards"
+  installFlagsToRemove: ""
+----
+
++
+Within a couple of minutes, the HyperShift Operator deployment in the `hypershift` namespace is updated to include the following environment variable:
+
++
+----
+    - name: MONITORING_DASHBOARDS
+      value: "1"
+----
+
++
+When monitoring dashboards are enabled, for each hosted cluster that the HyperShift Operator manages, the Operator creates a config map named `cp-[NAMESPACE]-[NAME]` in the `openshift-config-managed` namespace, where `NAMESPACE` is the namespace of the hosted cluster and `NAME` is the name of the hosted cluster. As a result, a new dashboard is added in the administrative console of the management cluster.
+
+. To view the dashboard, log in to the management cluster's console and go to the dashboard for the hosted cluster by clicking *Observe > Dashboards*.
+
+To disable a monitoring dashboards in a hosted cluster, remove the `--monitoring-dashboards` flag from the `hypershift-operator-install-flags` config map. When you delete a hosted cluster, its corresponding dashboard is also deleted.
+
+[#hosted-control-planes-customize-dashboards]
+== Customizing a monitoring dashboard
```

Comment:
I think as a user I might be a little confused to read this title that begins with a gerund and not see any formal steps/explicit description of what I could do. Is this section trying to say that you can go into the template file to change the values of the properties in the table? If so, maybe adding a sentence saying something like "you can edit these values to customize the dashboard" would be extra helpful.

Or if this section isn't meant to have any instructions, can the title be reworded to avoid starting with a gerund?

Reply:
I'll update the heading to no longer use a gerund.